enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,179 @@
+---
+title: customize-catalogs
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@jerolimov"
+  - "@cvogt"
+creation-date: 2022-07-27
+last-updated: 2022-08-23
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Catalogs
+
+## Summary
+
+The OpenShift Console provides a "Developer Catalog" which enables 
+the user to create apps and services based on Operator backed services, helm charts, templates, etc.
+
+Cluster admins can provide additional catalog items by installing operators or adding helm repositories.
+Cluster administrators want to hide these types.
+
+Under the developer catalog,
+there are categories and sub-categories. When users select a sub-category, they can see the list
+of types to install. This enhancement enable cluster admins to customize these types.
```

Comment:
The types aren't customizable per se, they can only be disabled, correct?

Reply:
Yes